```
FAIL: test_instantiate_all (testapps.test_pillowtop.tests.test_settings.PillowtopSettingsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/testapps/test_pillowtop/tests/test_settings.py", line 28, in test_instantiate_all
    self.assertEqual(len(all_pillow_configs), len(expected_meta))
AssertionError: 39 != 37
```

--- a/testapps/test_pillowtop/tests/data/all-pillow-meta.json
+++ b/testapps/test_pillowtop/tests/data/all-pillow-meta.json
@@ -55,6 +55,13 @@
         "full_class_name": "pillowtop.pillow.interface.ConstructedPillow",
         "name": "CaseSearchToElasticsearchPillow"
     },
+    "CouvertureFluffPillow": {
+        "advertised_name": "CouvertureFluffPillow",
+        "change_feed_type": "KafkaChangeFeed",
+        "checkpoint_id": "fluff.CouvertureFluffPillow.testhq",
+        "full_class_name": "custom.intrahealth.models.CouvertureFluffPillow",
+        "name": "CouvertureFluffPillow"
+    },
     "DefaultChangeFeedPillow": {
         "advertised_name": "DefaultChangeFeedPillow",
         "change_feed_type": "CouchChangeFeed",
@@ -96,6 +103,13 @@
         "checkpoint_id": "GroupToUserPillow-test_hqusers_2017-02-03",
         "full_class_name": "pillowtop.pillow.interface.ConstructedPillow",
         "name": "GroupToUserPillow"
+    },
+    "IntraHealthFluffPillow": {
+        "advertised_name": "IntraHealthFluffPillow",
+        "change_feed_type": "KafkaChangeFeed",
+        "checkpoint_id": "fluff.IntraHealthFluffPillow.testhq",
+        "full_class_name": "custom.intrahealth.models.IntraHealthFluffPillow",
+        "name": "IntraHealthFluffPillow"
     },
     "IntraHealthFormFluffPillow": {
         "advertised_name": "IntraHealthFormFluffPillow",

